chopsticks/xcm-transfer/shared.ts

```diff
@@ -0,0 +1,184 @@
+import { beforeEach, describe, it } from 'vitest'
+import { sendTransaction } from '@acala-network/chopsticks-testing'
+
+import { Network, NetworkNames, createContext, createNetworks } from '../../networks'
+import { check, checkEvents, checkHrmp, checkSystemEvents, checkUmp } from '../../helpers'
+
+import type { TestType as KusamaAssetHubTestType } from './kusama-assethub.test'
+import type { TestType as PolkadotAssetHubTestType } from './polkadot-assethub.test'
+
+type TestType =
+  | KusamaAssetHubTestType
+  | PolkadotAssetHubTestType
+
+export default function buildTest(tests: ReadonlyArray<TestType>) {
+  for (const { from, to, test, name, ...opt } of tests) {
+    describe(`'${from}' -> '${to}' xcm transfer '${name}'`, async () => {
```

Comment:
"xcm transfer" is really vague; it doesn't say what it's actually testing.

Reply:
This is the shared file that all tests use, I can rename but the idea is to keep all xcm related chopsticks tests there